llvm/docs/LangRef.rst

```diff
@@ -3577,6 +3577,12 @@ seq\_cst total orderings of other operations that are not marked
 Floating-Point Environment
 --------------------------
 
+Unless noted otherwise, LLVM works with IEEE-754 floating-point semantics: LLVM
+backends assume that the CPU is configured to provide IEEE-compatible behavior,
+and LLVM frontends can assume that LLVM IR floating-point operations behave
+according to the IEEE specification (with an :ref:`exception around NaN values
+<floatnan>`).
```

Comment:
Also the denormal exception

Reply:
What is the denormal exception? Is this about what happens when `denormal-fp-math` is set, but the default is to be IEEE-compatible?